.eslintignore

```diff
@@ -0,0 +1,38 @@
+# Don't check auto-generated stuff into git
+.parcel-cache/
+cypress-coverage/
+coverage/
+build/
+build-ext/
+build-dev/
+node_modules/
+stats.json
+.pnp
+.pnp.js
+generated/
+jest.config.js
+
+# misc
```

Comment:
> I would make it a link to explorer's gitignore

Simply using `.gitignore` won't suffice, since there are files that must be in git, but not linted. But now I have re-created this files based on explorer's gitignore.